v2/tools/generator/internal/astmodel/type_name.go

```diff
@@ -47,6 +46,7 @@ var armPackageDenyList = set.Make(
 	"insights",
 	"keyvault",
 	"kubernetesconfiguration",
+	"kusto",
```

Comment:
Is this just positioning for the upcoming PR?

Reply:
Smoothing the way for @super-harsh's upcoming PR, so he doesn't need to rebuild all the generated files.